### PR TITLE
Suggestion PBT initial ROCK

### PR DESCRIPTION
**Pull request by gboutry**

### CODE REVIEW

File: suggestion-pbt/rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    pbt:
+      plugin: dump
+      source: https://github.com/kubeflow/katib.git
+      source-depth: 1
+      source-type: git
+      source-subdir: cmd/suggestion/pbt/v1beta1
+      organize:
+        main.py: opt/katib/cmd/suggestion/pbt/v1beta1/main.py
+        requirements.txt: opt/katib/cmd/suggestion/pbt/v1beta1/requirements.txt
+      stage:
```
Comment (PietroPasotti) [inline suggestion-pbt/rockcraft.yaml:50]: not sure if staging these files is necessary

File: suggestion-pbt/rockcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        - opt/katib/cmd/suggestion/pbt/v1beta1/main.py
+        - opt/katib/cmd/suggestion/pbt/v1beta1/requirements.txt
```
Comment (PietroPasotti) [inline suggestion-pbt/rockcraft.yaml:52]: why put them in there when they could be in /usr/bin or some more memorable place?

File: suggestion-pbt/rockcraft.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    pbt-dependencies:
+      after: [pbt]
```
Comment (PietroPasotti) [inline suggestion-pbt/rockcraft.yaml:55]: don't think you need to wait on pbt for this step

File: suggestion-pbt/rockcraft.yaml (hunk 2 above)
Comment (gboutry) [reply suggestion-pbt/rockcraft.yaml:52]: This is the same place as the Dockerfile

File: suggestion-pbt/rockcraft.yaml (hunk 1 above)
Comment (gboutry) [reply suggestion-pbt/rockcraft.yaml:50]: I had an issue where the files were not staged anymore, I will try again without, I might have resolved it

Comment (PietroPasotti) [reply suggestion-pbt/rockcraft.yaml:50]: I asked and it appears that it is necessary after all

**Comment by i-chvets** [comment]
This PR was used as a based for final soltion https://github.com/canonical/katib-rocks/pull/24
Closing.

**Pull request closed without merging**